.ddev/.typo3-setup/scripts/utils.sh

```
@@ -0,0 +1,316 @@
+    fi
+    mysql -uroot -proot -e "DROP DATABASE IF EXISTS $DATABASE"
+}
+
+# Function to create symlinks for the main extension.
+# It iterates over the items in the current directory, excluding certain directories and files,
+# and creates symbolic links for the remaining items in the specified base path.
+function create_symlinks_main_extension() {
+    local EXCLUSIONS_ARRAY=()
+    IFS=' ' read -r -a EXCLUSIONS_ARRAY <<< "$EXCLUSIONS"
+    for item in ./*; do
+        local base_name=$(basename "$item")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)